git_machete/cmd.py

```diff
@@ -1254,7 +1254,9 @@ def delete_unmanaged(self) -> None:
             print("No branches to delete")
 
     def edit(self) -> int:
-        return run_cmd(self.cli_ctxt, get_default_editor(self), self.definition_file_path)
+        if self.definition_file_path is None or self.definition_file_path == "":
```

Comment:
Hmm is there any valid execution path where `definition_file_path` ends up `None` or `""`, other than the invocation of `git machete help`? 🤔

Reply:
Actually! I think it makes sense to set `self.definition_file_path` in c'tor of this class rather than in `launch` function